sonar-update-center-common/src/test/java/org/sonar/updatecenter/common/VersionTest.java

```diff
@@ -160,6 +160,15 @@ public void testPatchFields() {
     assertThat(version.equals(Version.create("1.2.3.5"))).isFalse();
   }
 
+  @Test
+  public void isPatchVersion_whenIsPatchVersion_returnTrue() {
```

Comment:
It would be cool to have this test checking both cases, by passing a matrix with a patch and a non patch version and the expected outcome of the isPatchVersion result.

Reply:
Added second case.